client/my-sites/domains/domain-management/dns/add-dns-record.jsx

```diff
@@ -97,24 +109,25 @@ class AddDnsRecprd extends Component {
 				},
 			}
 		);
+		const recordBeingEdited = this.getRecordBeingEdited();
+		const headerText = recordBeingEdited
+			? translate( 'Edit DNS record' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Edit DNS Record' )` *ES Score: 11*